test/dynamo/test_logging.py

```diff
@@ -666,6 +666,20 @@ def fn(a):
             len([r for r in records if "return a + 1" in r.getMessage()]), 0
         )
 
+    @requires_cuda
+    @make_logging_test
+    def test_compile_debug(self, records):
```

Comment:
Could you move this test to `test/_inductor/test_cuda_repro.py` since this is an inductor issue?

Reply:
@peterbell10 should be resolved :) have a nice weekend